config.go

```diff
@@ -111,6 +114,8 @@ func GetStorageReader(config Config) (storageconnector.StorageReader, error) {
 		return storageconnector.NewMemoryStorage(data), nil // implement a generator of random list of features?
 	case StorageTypeGit:
 		return storageconnector.NewGitStorage(config.Git.URL)
+	case StorageTypePG:
+		return storageconnector.NewPostgresStorage("omaurer", "pazuzu") // FIXME: this is for testing, it needs to be properly put in the config file
```

Comment:
Yes we should and we should discuss where to put it. Probably somewhere in .pazuzu-cli.yaml